apm-agent-plugins/apm-jaxrs-plugin/src/main/java/co/elastic/apm/agent/jaxrs/JaxRsTransactionNameInstrumentation.java

```diff
@@ -79,6 +89,13 @@ private static void setTransactionName(@SimpleMethodSignature String signature,
                     }
                 }
                 transaction.withName(transactionName, PRIO_HIGH_LEVEL_FRAMEWORK, false);
+                transaction.getContext().setFrameworkName(FRAMEWORK_NAME);
+                String version = versionsCache.get(javax.ws.rs.GET.class);
+                if (version == null) {
+                    version = VersionUtils.getVersionFromPomProperties(javax.ws.rs.GET.class, GROUP_ID, ARTIFACT_ID);
```

Comment:
`VersionUtils` may return `null`. Avoid re-evaluating in this case.

Reply:
fixed